### DISCUSSION TITLE
SteamVR only working on Firefox

### DISCUSSION THREAD

**Discussion by lihong-xmuseum** [mozilla/hubs#4893] [Hubs Cloud Questions] [answered]
Guys, today when I connected VR, I found that Google and Edge appeared as screenshots. I can't directly enter VR, only Firefox can. Steam VR 1.20.
![1638769438(1)](https://user-images.githubusercontent.com/69456381/144815640-c755bdf9-7fe1-4f7d-af70-9afd57a9e307.png)
 4

**Comment by pattersonbl2** [comment, marked as answer]
I have opened an issue around this.  I don't have a eta on when this will be completed

**Reply by lihong-xmuseum** [reply]
ok，thanks，I'll stay tuned

**Reply by matthewbcool** [reply]
Does this advice help? https://github.com/mozilla/hubs/issues/5749#issuecomment-1499036105
